web/templates/try_zulip_modal.hbs

```diff
@@ -0,0 +1,29 @@
+<p>
+    {{t "Explore how hundreds of community participants use Zulip to brainstorm ideas, discuss technical challenges, ask questions, and give feedback:" }}
+
+    <ul>
+        <li>
+            {{#tr}}
+            You'll see a list of <b>recent conversations</b>, where each conversation is
+            labeled with a topic by the person who started it. Click on a conversation to
+            view it. You can always get back to recent conversations from the left sidebar.
+            {{/tr}}
+        </li>
+        <li>
+            {{#tr}}
+            Click the name of a channel in the left sidebar, and click on any topic underneath
+            to view one conversation at a time. You can explore discussions of changes to the
+            design of the Zulip app in <z-design-channel>#design</z-design-channel>, or see ongoing
+            issue investigations in <z-issues-channel>#issues</z-issues-channel>.
+            {{#*inline "z-design-channel"}}<a href="/#narrow/channel/101-design">{{> @partial-block}}</a>{{/inline}}
+            {{#*inline "z-issues-channel"}}<a href="/#narrow/channel/9-issues">{{> @partial-block}}</a>{{/inline}}
+            {{/tr}}
+        </li>
```

Comment:
These are channel feed links, I think? We should think about whether we want them to work that way or have JS code to take you to the first topic.

Reply:
Let's just make these bold-formatted rather than being links; I don't think we need to link them here.

Reply:
Made it bold. See updated screenshot in the PR description.